### ISSUE TITLE
[Participate Page] Pulse section

### ISSUE THREAD

**Issue by taisdesouzalessa** [mozilla/foundation.mozilla.org#1641]
**Description:** during a design crit session for the [Participate Page](https://redpen.io/kc1c8298f2ce858512), Matthew brought up the idea of including a Pulse section since there are lots of opportunities for contributions there, from attending to an event to writing documentation. People who land on this page are in the mindset of acting so it seems a good opportunity to include Pulse as well, with the proper framing.

We started with 2 simple components: one explains what Pulse is and links to the site and the other surfaced the type of help needed:
**Simple component - option 1**
<img width="592" alt="image" src="https://user-images.githubusercontent.com/22157921/42117987-d2aa0978-7bb4-11e8-9fc2-95f63af8f178.png">

**Simple component - option 2**
<img width="601" alt="image" src="https://user-images.githubusercontent.com/22157921/42118049-43de58f6-7bb5-11e8-802e-7e0781f4d6ce.png">

In a chat with Matthew, we found there is an opportunity to open the invitation to contribute to Pulse even more, by emphasizing in the component that not only designers and coders can participate on projects. Help is needed from very simple tasks such as test and feedback for instance. Based on that, I worked on 2 more iterations - posted here.

**Comment by taisdesouzalessa** [comment]
### Option 1: 
having all the tags displayed in the component and clickable. For instance, if user clicks on tag `Take Action` they will be taken to [this link](https://www.mozillapulse.org/help/take-action). 

<img width="1260" alt="image" src="https://user-images.githubusercontent.com/22157921/42118172-1cadbdd4-7bb6-11e8-9a85-cb04b51eb082.png">


### Option 2: 
list view. All the opportunities to help are displayed in a subtle way and non-clickable. If audience wants to see more projects they will be taken to Pulse main page through the button. There is a small friction on this flow since once user lands on the main Pulse page, they have to click in one of the cards to identify the type of help needed.

<img width="1267" alt="image" src="https://user-images.githubusercontent.com/22157921/42118254-a87d58ba-7bb6-11e8-9e57-06e77805c3d3.png">

InVision here **(scroll all the way down to see component at the bottom)**  [Option 1](https://mozilla.invisionapp.com/share/8YM8464ZDA2#/305937301_proposal_4_-_Ux_Flow_-_Main_Copy) | [Option 2](https://mozilla.invisionapp.com/share/SHM846QPVTJ#/306221243_proposal_4_-_Ux_Flow_-_Pulse_Component_Opt_1)

@natalieworth @kristinashu @xmatthewx please give me your feedback, feel free to offer other options if you have any other solution in mind!

**Comment by xmatthewx** [comment]
I added a bunch of comments on Option 2 in InVision. Ping me with any questions.

**Comment by taisdesouzalessa** [comment]
Based on your Matthew's feedback, here is the revised comp:
<img width="1208" alt="image" src="https://user-images.githubusercontent.com/22157921/42248900-e563f580-7edc-11e8-85db-3fea26328907.png">
InVision [here](https://mozilla.invisionapp.com/share/ERMG8NHF3DH#/306768513_LATEST_ITERATION_-_proposal_4_-_Ux_Flow_-_Pulse_Component_Opt_1_Copy)

Questions for Engineers regarding implementation possibilities in this ticket: #1650

**Comment by taisdesouzalessa** [comment]
Based on design crit, here is the new iteration :).
<img width="1242" alt="image" src="https://user-images.githubusercontent.com/22157921/42403972-230b6724-813a-11e8-9474-377a9da149eb.png">

InVision [here](https://mozilla.invisionapp.com/share/VRMPX2RT9BX#/307396738_LATEST_ITERATION_-_proposal_4_-_Ux_Flow_-_Pulse_Component_-_Latest_Iteration_V2)

**Comment by kristinashu** [comment]
Is design finalized for this? If so can you please close this design ticket and open an implementation ticket up with exact requirements and specs for devs? But since this is P2, don't assign it to anyone yet.

**Comment by taisdesouzalessa** [comment]
I think design is good to go. It has been through some rounds of critique already. @xmatthewx you cool with the[ latest version ](https://github.com/mozilla/foundation.mozilla.org/issues/1641#issuecomment-403170232) of it? If so, let me know so I can write the specs.

**Comment by xmatthewx** [comment]
If mobile is ready, I think it's good to go! One nitpick: the line break on the left before "Help with your time..." is awkward. Delete the break for one paragraph or add a paragraph space. 

For implementation, note
 - I worked with Gideon to add a helpful api route `/v2/entries/?has_help_types=True`. https://github.com/mozilla/foundation.mozilla.org/issues/1650#issuecomment-410033031
 - Total project count might be a nice to have. If we can't, might fallback to: "There are nearly 1000 projects in Pulse!"

I'm glad to peek at the requirements checklist when it's ready.

**Comment by taisdesouzalessa** [comment]
Good catch and I'll work on the mobile/specs tomorrow 🎉

**Comment by taisdesouzalessa** [comment]
@beccaklam I made some changes on the design of the section "Collaborate with us". Could you please take a look at the file in Abstract and make sure the UI and spacings are correct (branch: "Pulse-Participate Page Mobile", section "Pulse section iterations + Wins")? I did my best to follow the guidelines that you set for the rest of the page but I think it would be good if you double check it since you built the rest of the UI.

Here are the main changes:
1. for mobile
• Increased the spacing between the items of the list (48px of tap area). Reason: if they are too close as we have it in desktop, users will have a hard time tapping on individual elements. Apple guidelines are 44px and I am using 48px to keep our rule of multiples of 8px).
• Kept the same proportion as we have on desktop for the Pulse cards. Reason: easy to implement (Pulse cards pictures don't have to be cropped again for mobile and it follows the same standard as we have in our home page "Trending Projects" section).
• Adjusted the CTA size (same as the other buttons on the page, they maintain their size on mobile - I think the only button that gets smaller is the "donate" on the top menu).

2. for tablet:
• Increased the spacing between the items of the list (48px of tap area).
• Kept the same proportion as we have on desktop for the Pulse cards, which caused some little changes in the layout.

Let me know if you have any questions and we can chat at anytime!

**Comment by beccaklam** [comment]
@taisdesouzalessa Component is reviewed 👍 I updated the pacing to match bootstrap and merged your branch. You can check out the changes on Master's Initiatives and Participate file :)

**Comment by taisdesouzalessa** [comment]
It looks great, Becca! Thanks for merging it and reviewing spaces! So I guess there is no action item for me here anymore, right?

**Comment by beccaklam** [comment]
Not that I know of! I think we're good to close if you are good with that too :)

**Comment by taisdesouzalessa** [comment]
I'll keep it open until I do the requirements checklist then I can close it, okay?

**Comment by taisdesouzalessa** [comment]
@beccaklam implementation ticket is here:  #1731

Please take a look and let me know if there is anything missing there :).
